examples/viewer.html

```diff
@@ -194,7 +194,8 @@
         paddingTop: this.header./*OK*/offsetHeight,
         visibilityState: this.visibilityState_,
         prerenderSize: this.prerenderSize_,
-        viewerorigin: parseUrl(window.location.href).origin
+        viewerorigin: parseUrl(window.location.href).origin,
+        csi: 0
```

Comment:
done.

added some additional code in `performance.js` to ignore all `ticks` and `flush` calls when performanceTracking is off.